assets/prototype/domain/eventEditor/containers/queries/useInitQueries.js

```diff
@@ -1,4 +1,5 @@
 import get from 'lodash/get';
```

Comment:
should remove this import if it is no longer being used

Reply:
Sure thing, thanks, updated.